package.json

```diff
@@ -11,14 +11,15 @@
     "gui:devtools": "yarn workspace @ethui/gui devtools",
     "storybook": "yarn workspace @ethui/ui storybook",
     "ext:dev": "yarn workspace @ethui/extension dev",
+    "ext:dev:all": "yarn ext:dev --target chrome-dev && yarn ext:dev --target firefox-dev",
```

Comment:
this should be a script on `extension/package.json`, not here
and ideally it would run both in parallel, otherwise you're adding delay for us firefox users

Reply:
I thought this was a better place for it, but I'll include it in the package instead. Any suggestions for running it in parallel? I think the delay applies to both chrome and firefox users since both will wait for the script to run, otherwise you can just run each (firefox or chrome) individually.